src/seahawk/seahawk_deck/thrust.py

```diff
@@ -39,28 +44,223 @@ class Thrust(Node):
     def __init__(self):
         """Initialize this node"""
         super().__init__('thrust')
+
+        self.MAX_FWD_THRUST = 36.3826715 # N
+        self.MAX_REV_THRUST = -28.6354180 # N
+
+        self.TOTAL_CURRENT_LIMIT = 70 # A
+        self.ESC_CURRENT_LIMIT = 40 # A
+
+        self.motor_positions = [
+            [0.19, 0.12, 0.047],
+            [0.19, -0.12, 0.047],
+            [-0.19, 0.12, 0.047],
+            [-0.19, -0.12, 0.047],
+            [0.105, 0.15, -0.038],
+            [0.105, -0.15, -0.038],
+            [-0.105, 0.15, -0.038],
+            [-0.105, -0.15, -0.038]
+        ]
+
+        self.motor_thrusts = [
```

Comment:
Consider adding a comment on this and `self.motor_positions` elaborating where the numbers are from/what they represent